lib/commands/connect/shared.js

```diff
@@ -172,7 +172,7 @@ exports.withConnection = function(context, heroku) {
         console.log("No connection found");
         reject();
       } else if (connections.length > 1) {
-        console.log("Multiple connections found");
+        console.log("Multiple connections found. Please use '--resource' to specify a single connection by resource name.");
```

Comment:
Because I moved over to using this function, I didn't want to use the extra info about using the `--resource` flag. Also, I'm planning to go back and clean up the flag usage a bit to remove the use of `-r`, because it conflicts with the toolbelt's own `--remote` flag. I'll do that in another PR, but that's why I removed the `-r` from this message.